packages/xrpl/test/client/constructor.test.ts

```diff
@@ -1,24 +1,25 @@
 import { assert } from 'chai'
-import { Client } from 'xrpl-local'
+
+import { Client } from '../../src/client'
```

Comment:
Why did this go back to `../../src/client` instead of `xrpl-local`?

Reply:
Xrpl local was needed with mocha but not now. Using xrpl local resulted in duplicate typescript definition files that caused errors in typescript checking.

improring from XRPL local versus a local file caused a mismatch between the library src files and the test files themselves using the same interfaces.